src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/util/Utils.java

```diff
@@ -1528,8 +1525,9 @@ private DocCollator(Locale locale, int strength) {
             instance = createCollator(locale);
             instance.setStrength(strength);
 
-            keys = new LinkedHashMap<String, CollationKey>(MAX_SIZE + 1, 0.75f, true) {
+            keys = new LinkedHashMap<>(MAX_SIZE + 1, 0.75f, true) {
```

Comment:
This map's initial capacity requires it to resize before it starts evicting, see https://github.com/openjdk/jdk/blob/7ebc4bce930e7d406f1e68c8bd09ac1dc3a21e5a/src/java.base/share/classes/java/util/HashMap.java#L2563

Reply:
Not in this PR.